cylc/rose/entry_points.py

```diff
@@ -222,3 +228,44 @@ def record_cylc_install_options(
     )
     dumper(rose_suite_conf, rose_conf_filepath)
     return cli_config, rose_suite_conf
+
+
+def copy_config_file(
+    dir_=None,
```

Comment:
Worth changing `_dir` to `srcdir` and `dest_root` to `destdir` for consistency with : https://github.com/cylc/cylc-rose/pull/40

Reply:
No - these have to be the same for all the entry point functions and require a change in Cylc-flow to match, so doing this here would break things. 
I will be rebasing #40 once everything else is in.